### ISSUE TITLE
ADD option to display values in multiple lines

### ISSUE THREAD

**Issue by allanlaal** [asafh/chrome-tabcounter#4]
## use case

currently I have only 260 tabs open in 58 windows.
I Click on the extension icon to toggle between views: I choose the "Tab count/Window Count" view

## expected result

the extension icon shows: 260/58

## actual result

the extension icon shows: 260/: (50% of the 5 is visible)

## solution

since the extension icon itself is not important to users, please add an option to display values in multiple lines. Ideally 3 lines to support #5  aswell

### example

-------------
260
[58]
(3)

-------------

where (3) is the number of tabs opened in current window (if #5 gets supported)

**Comment by asafh** [comment]
Unforunately Chrome doesn't allow new lines in badge texts. Morover the badge text is [limited](https://developer.chrome.com/extensions/browserAction#badge) to 4 characters and there isn't much around it.  
The only place we could could show more info is with either the tooltip that appears when hovering (i.e. the "Tab count/Window Count" text), or a popup that opens on click. I feel like a popup is more usable as the hover tooltip is rather slow to appear. I'll try and get around to it this week.

 Cheers

**Comment by allanlaal** [comment]
damn Chrome and its limitations :(

is it possible to dynamically change the extension icon to a generated image with all the needed numbers to bypass this?

**Comment by asafh** [comment]
Yeah good idea, that'll work. I've created a working proof of concept but I need to see how can I make it aesthetic/readable, and optionally customizable.
